Snakefile-sorting-hat

```diff
@@ -187,7 +187,7 @@ rule produce_recalculations_call:
         pushd {input.working_directory}
 
 
-        snakemake --latency-wait 100 {config['sm_options']} --config \
+        snakemake --forceall --latency-wait 150 {config['sm_options']} --config \
```

Comment:
`--forceall` needs to be optional, so that if a variable is set, then it runs with it. We need to be able to turn this on and off. It will only be used like this on the first run of the release, then subsequent runs of the same release shouldn't have to re-run everything.